StartLearningHere.md

```diff
@@ -60,3 +60,4 @@ This demo is heavily inspired by [Sipke Schoorstra's Orchard Harvest session](ht
 * [Unit and integration testing](Services/TestedService.cs)
 * [Writing Vue.js applications in Orchard Core modules](Controllers/VueJsController.cs)
 * [Compiling resources using Gulp](Gulpfile.js)
+* [GraphQL](GraphQL/Startup.cs)

```

Comment:
Ezt a Gulpfile végén hivatkozni is kellene, lásd: https://github.com/Lombiq/Orchard-Training-Demo-Module/#contributing-and-support (meg a befejezést onnan átrakni).